app/views/costs/CostsSummaryPage.scala.html

```diff
@@ -0,0 +1,157 @@
+                            <label for="search" class="form-label">Name</label>
+                        </dt>
+                        <dd>
+                            <input class="search form-control" id="search" type="search" name="name" value='@form("name").value' autofocus>
+                        </dd>
+                    </dl>
+                </div>
+                <div class="col-md-3">
+                @select(
+                    field = form("team"),
+                    options = teams.map(t => t.name.asString -> t.name.asString),
+                    Symbol("_default") -> "All",
+                    Symbol("_label") -> "Team",
+                    Symbol("_labelClass") -> "form-label",
+                    Symbol("id") -> "team-filter",
+                    Symbol("class") -> "form-select"
+                )
+                </div>
+            </div>
+        </form>
+
+        <table class="table table-striped sticky-header table-layout: fixed" id="repo-list">
+            <thead>
+                <tr>
+                    <th rowspan="2"><button role="button" id="name" data-sort="repo-name" class="fw-bold sort no-border">
+                        Application Name</button></th>
+                    <th colspan="2" style="text-align: center">Integration</th>
+                    <th colspan="2" style="text-align: center">Development</th>
+                    <th colspan="2" style="text-align: center">QA</th>
+                    <th colspan="2" style="text-align: center">Staging</th>
+                    <th colspan="2" style="text-align: center">External Test</th>
+                    <th colspan="2" style="text-align: center">Production</th>
+                    <th rowspan="2">
+                        <button role="button" id="yearlyCost" data-sort="yearly-cost" class="fw-bold sort no-border">
+                            Estimate Cost (yearly)
+                        </button>
+                    </th>
+                </tr>
+                <tr>
+                    <th>Slots</th>
+                    <th>Instances</th>
+                    <th>Slots</th>
+                    <th>Instances</th>
+                    <th>Slots</th>
+                    <th>Instances</th>
+                    <th>Slots</th>
+                    <th>Instances</th>
+                    <th>Slots</th>
+                    <th>Instances</th>
+                    <th>Slots</th>
+                    <th>Instances</th>
+                </tr>
+            </thead>
+            <tbody class="list" id="matches">
+            @configs.zipWithIndex.map { case (config, index) =>
+            <tr>
+                <td class="repo-name" id="row@{
+                    index
+                }_name"><a id="link-to-@config.serviceName-page" href="/repositories/@config.serviceName">@config.serviceName</a></td>
+                <td style="text-align: center">@config.getIntegrationEnvironment.map(_.deploymentSize.slots)</td>    @* Integration - slots *@
```

Comment:
We should iterate over `Environment.values` - this already has a predefined order (examples on other pages with env columns). It would reduce the code - and the use of a single extractor for all envs even more useful. `@config.deploymentConfigs(env).map(_.deploymentSize.slots)`

Since we're only ever interested in deploymentSize - the extractor could even be `@config.deploymentSize(env)`.
Infact, does `ServiceAndEnvironment` even need to keep `Seq[DeploymentConfig]`? Maybe just `Map[Environment, DeploymentSize]`

The comment seems unnecessary.